classes/PodsMeta.php

```diff
@@ -285,6 +289,7 @@ public function core() {
 				if ( ! pods_tableless() ) {
 					add_filter( 'add_comment_metadata', array( $this, 'add_comment_meta' ), 10, 5 );
 					add_filter( 'update_comment_metadata', array( $this, 'update_comment_meta' ), 10, 5 );
+					add_filter( 'update_comment_metadata_by_id', array( $this, 'update_comment_meta_by_id' ), 10, 4 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 1816561c9c397abe7e73758a436b49ba</sub>